### PR TITLE
Changed icon color to improve general UX

### LINKED ISSUE #12844: [beta][UI][accessibility] icon color change for challenges
The icon color for the challenges, testing div
`div.challenge-test-suite`
has a color that is not accessible at` #d9edf7`

Please change to the default color of `darkgreen`

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #12844

#### Description

Changed icon color to be better appealing and less distracting to the user. Was previously too light, then changed to green which was mis-informative, now it is dark grey similar to color of the text (see image below). 
screen shot 2017-03-03 at 11 07 50 am

### CODE REVIEW

**Comment by Bouncey** [comment]
@Ethan-Arrowood can you show me what is looks like in night mode? I do like the grey

File: client/less/challenge.less [hunk 1]
```diff
@@ -138,7 +138,7 @@
 }
 
 .refresh-icon {
-  color: darkgreen;
+  color: #575757;
```
Comment (raisedadead) [inline client/less/challenge.less]: please declare this as a variable and use here.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: client/less/challenge.less (hunk 1 above)
Comment (Ethan-Arrowood) [inline client/less/challenge.less]: Okay

Comment (Ethan-Arrowood) [inline client/less/challenge.less]: Once I make the change whats the correct way to fix my PR?

Comment (raisedadead) [inline client/less/challenge.less]: Follow the below:

```sh
git add <filename>
git commit --amend --no-edit
git push --force
```

Comment (raisedadead) [inline client/less/challenge.less]: This will automatically update the PR.

**Comment by renjit00** [comment]
Hi guys....i want to join with you ...

File: client/less/challenge.less (hunk 1 above)
Comment (Ethan-Arrowood) [inline client/less/challenge.less]: Okay great thank you.

**Comment by systimotic** [comment]
@Bouncey Very good point. This will probably need to be changed in the night mode.  

firefox_2017-03-03_23-49-55

File: client/less/challenge.less (hunk 1 above)
Comment (Ethan-Arrowood) [inline client/less/challenge.less]: @raisedadead 

> please declare this as a variable and use here.

Is there a preferred file I should declare the variable or can I do it in the challenge.less file?

Comment (BerkeleyTrue) [inline client/less/challenge.less]: client/less/bootstrap/variables

Comment (Ethan-Arrowood) [inline client/less/challenge.less]: thank you @BerkeleyTrue didn't know if I was allowed to make edits there or not.

**Comment by Ethan-Arrowood** [comment]
@systimotic How does this look for night mode?
screen shot 2017-03-04 at 5 11 24 pm
I made it match the font color

**Comment by Bouncey** [comment]
## LGTM :+1: 

Thanks for contributing :tada:

**Comment by Ethan-Arrowood** [comment]
@Bouncey Thank you! I love working on FCC 😄

**Comment by systimotic** [comment]
@Ethan-Arrowood Thanks a lot for contributing! Isn't it so much fun?

**Comment by Ethan-Arrowood** [comment]
@systimotic It is! I'm already working on another issue regarding the npm tests 😋

**Pull request merged**